### DISCUSSION TITLE
Approximate release date for v6.0.0

### DISCUSSION THREAD

**Discussion by edgarrmondragon** [copier-org/copier#526] [General]
Hello and thank you for working on `copier`!

I've been PoCing converting an existing cookiecutter template to copier to leverage migrations (killer feature!). However, another one that I'd be missing from cookiecutter is support for [Jinja extensions](https://github.com/copier-org/copier/pull/370). I know they're currently supported in `master`, but is there a due date for the release of `v6.0.0`?

**Comment by yajo** [comment]
Not for now, sorry. In https://github.com/copier-org/copier/milestones/v6.0.0 you can see what's missing. Recently I've reduced the scope to be able to release sooner. Once that's clear, I'll publish first beta and ask for community feedback.

Any contribution in testing, money or code is welcome. 😊
